lib/wit/viceroy.wit

```diff
@@ -0,0 +1,5 @@
+package fastly:viceroy;
+
+world compute {
+	include fastly:api/compute;
```

Comment:
I'm a little surprised not to see some of the basic stuff, like the CLI I/O stuff here. I'm OK that it's not, but just wanted to put something here so you think about whether you want to add in some of the basic, easy wins here or not. Feel free to resolve immediately after reading this paragraph, though, if you'd already thought of it.

Reply:
This is definitely something that requires more documentation, as the world that we want to run components in for viceroy will be larger than the world that's targeted by SDK code. This gives the mocks we write the ability to interact with the local fs or sockets even, but still prevents customer components from doing the same.